contents.json

```diff
@@ -16439,6 +16439,33 @@
       ],
       "date_added": "May 26 2018",
       "suggested_by": "@vpeschenkov"
-    }
-  ]
+    },
+    {
+      "title": "FityIt - The shapes game",
```

Comment:
Could you remove this ` - The shapes game` from the title?

Reply:
Ok